en/editor/scene/index.md

```diff
@@ -0,0 +1,44 @@
+# Scene Editor
+<img src="images/scene-editor.png" width="100%" height="100%"/>
+
+**Scene Editor** is the core working area of content creation, You will use it to choose an place the scene image, character, effect, UI and other game elements. in this working area, you can select and use **transform tool** to change the position, rotation, scale, size and other attributes of node, also you will preview the WYSIWYG scene.
+
+## View Introduction
+### Navigation
+There are some differences between 3D view and 2D view navigation. you can change between 3D view and 2D view by click the 3D/2D button on the toolbar.3D view is used for 3D scene editing，2D is used for UI,Sprite editing.
```

Comment:
There are some differences between 3D view and 2D view navigation. you can change between 3D view and 2D view by click the 3D/2D button on the toolbar.3D view is used for 3D scene editing，2D is used for UI,Sprite editing.
-->
There are some differences between 3D view and 2D view navigation. You can change between 3D view and 2D view by clicking the __3D/2D button__ on the toolbar. 3D view is used for 3D scene editing，2D is used for UI, Sprite editing.